lib/ttfunk/subset/mac_roman.rb

```diff
@@ -11,36 +10,41 @@ def initialize(original)
         @subset = Array.new(256)
       end
 
+      def code_page
+        'mac_roman'.freeze
+      end
+
       def to_unicode_map
-        Encoding::MacRoman::TO_UNICODE
+        CodePages[code_page].unicode_mapping
       end
 
       def use(character)
-        @subset[Encoding::MacRoman::FROM_UNICODE[character]] = character
+        @subset[from_unicode(character)] = character
       end
 
       def covers?(character)
-        Encoding::MacRoman.covers?(character)
+        !from_unicode(character).nil?
       end
 
       def includes?(character)
-        code = Encoding::MacRoman::FROM_UNICODE[character]
+        code = from_unicode(character)
         code && @subset[code]
       end
 
       def from_unicode(character)
-        Encoding::MacRoman::FROM_UNICODE[character]
+        CodePages[code_page].code_page_mapping[character]
       end
 
-      protected
+      def new_cmap_table(_options = nil)
```

Comment:
I guess the optional argument can be dropped as it seem to be never used. In all `Subset` classes.

Reply:
Yes, good idea.